site/en/docs/extensions/migrating/publish-mv3/index.md

```diff
@@ -0,0 +1,160 @@
+        <figcaption>
+            Trusted tester emails in the Chrome Web Store developer Account page.
+        </figcaption>
+    </figure>
+1. Save changes at the bottom of the page.
+1. [Upload][cws-upload] the beta version of your extension
+1. Go to the Distribution tab
+1. Set the Visibility to **Private**. 
+
+<figure>
+    {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/EfL45p0rsDFExxbjMp0B.png", alt="Setting the Visibility as Private in the Distribution tab", width="500", height="612", class='screenshot' %}
+    <figcaption>
+    Visibility set to Private in the Distribution tab. 
+    </figcaption>
+</figure>
+
+### Distribute to members of a Google group {: #dist-group }
+
+Once you've received enough feedback from your beta testers, you can expand distribution to members of a [Google group][google-group] that you own or manage. 
+
+Go to the Distribution tab, set the Visibility to **Private**, and choose your beta tester Google group from the dropdown menu. 
+
+<figure>
+    {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/eW95QyUHVYrQw0eThSNc.png", alt="ALT_TEXT_HERE", width="600", height="445" %}
+    <figcaption>
+        Distributing to Members of a Google Group
+    </figcaption>
+</figure>
+
+### Distribute to testers with a direct link {: #dist-link }
+
+Another alternative is to set the visibility to **Unlisted**. This way, only users who have the direct link can install the extension.
+
+## Gradually roll-out your release {: #gradual-rollout }
+
+To ensure that any unexpected problems will have minimal impact, you can roll out your update gradually by following these steps. This is only available for extensions with more than 10,000 active users.
+
+1. [Upload][cws-upload] your new version.
+2. Go to the Distribution tab.
+3. Enter a percentage in **Percentage rollout** field.
+
+<figure>
+    {% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/yoMNFt1ht6qSLXzFyrWj.png", alt="Screenshot showing the 'more' menu's defer publish option", width="286", height="184", class='screenshot' %}
+    <figcaption>
+        Setting the initial rollout percentage.
+    </figcaption>
+</figure>
+
+To continue rolling out gradually, navigate to the **Package** tab of your item and increase the percentage located in the **Published** section. Note that the percentage can only be increased, and it doesn't trigger an additional review.
+
+<figure>
+    {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/UA8vDp8aG89exqqjzY55.png", alt="Screenshot of the Chrome Web Store update percent rollout field", width="800", height="395", class='screenshot' %}
+    <figcaption>
+        Updating the rollout percentage
+    </figcaption>
+</figure>
+
+## Plan for review times {: #review }
+
+We recommend planning enough time for your item to be reviewed as review times may vary based on [different factors][review-factors]. Consider staging your release and regularly checking your item's status to make changes quickly if necessary.
```

Comment:
Is it possible to provide a minimum number?

Reply:
That was my reaction on first read, when I didn't know that there was a number farther down. I still think this is too vague, but I'm not sure what to do about it.

Reply:
I rewrote this a bit. LMKWYT